SpeechToText: Add API descriptions (#8048)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxSpeechToText/1 Configuration/onContentReady.md

---
---
##### shortDescription

> A function that is executed when the UI component is rendered and each time the component is repainted.

Reviewer: ```suggestion
A function that is executed when the UI component is rendered or repainted.
```
